web-tester/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Change Log (web-tester)
 
+## [10.0.16] - 2024-04-17
+
+### Added
+
+- Add EngineUrl#queryParam method: This was needed as the the change in 10.0.14 encoded query parameter if they are added via the EngineUrl#path method.
```

Comment:
the sentence doesn't make sense: Not sure what you wanted to say; but this would be my proposal:

This was needed due to the change in 10.0.4, which encoded query parameters if they were added via the EngineUrl#path method.